examples/xml-from-record-conversion/xml_from_record_conversion.bal

```diff
@@ -0,0 +1,48 @@
+import ballerina/io;
+import ballerina/xmldata;
+
+// Defines a record type with annotations.
+@xmldata:Namespace {
+    prefix: "ns",
+    uri: "http://sdf.com"
+}
+type Invoice record {
+    int id;
+    Item[] items;
+    @xmldata:Attribute
+    string 'xmlns?;
+    @xmldata:Attribute
+    string attr?;
+};
+
+@xmldata:Namespace {
+    uri: "http://example.com"
+}
+@xmldata:Name {
+    value: "purchased_item"
+}
+type Item record {
+    string itemCode;
+    @xmldata:Name {
+        value: "item_count"
+    }
+    int count;
+};
```

Comment:
Shall we remove Name annotation from the example

Reply:
Removed [6999224](https://github.com/ballerina-platform/ballerina-distribution/pull/2963/commits/69992249f0ae1503bb90d6bcaa0f083fba511e7e)